Utils/ExtractVersionFromTilsesetJSON/compute_versiondata_from_py3dfiles.py

```diff
@@ -96,8 +109,8 @@ def get_featuresid(transactions, millesime):
 """
 # Iterrate on a Dataframe to extract id of transactions for the versionTransation and id of buildings for version
 #
-# @Param:
-#   df : DataFrame
+# @Params:
+#   df : DataFrame (columns=["id", "startDate", "endDate", "source", "destination", "type", "transactions"])
```

Comment:
Thanks for adding the columns. Could you specify what's in this dataframe please ? Is it all buildings and their transactions ? Is it only the buildings of a specific year ? Something else ?